projects/dsp-ui/src/lib/action/components/confirmation-dialog/confirmation-dialog.component.spec.ts

```diff
@@ -99,10 +110,19 @@ describe('ConfirmationDialogComponent', () => {
             expect(dialogDiv).toBeTruthy();
 
             const dialogTitle = dialogDiv.querySelector('.title');
-            expect(dialogTitle.innerHTML.trim()).toEqual('Title');
+            expect(dialogTitle.innerHTML.trim()).toEqual('Are you sure you want to delete this value from My label?');
+
+            const dialogMessageLabel = dialogDiv.querySelector('.message .val-label');
+            expect(dialogMessageLabel.innerHTML.trim()).toEqual('Confirming this action will delete the following value from My label:');
```

Comment:
this comes from `ConfirmationMessageComponent`, right?

Reply:
Correct. Actually, I think testing for the message component contents is a bit redundant here. It should just be enough to test the message component contents in the `ConfirmationMessageComponent` spec file, right?

Reply:
Yes, I think so. I think you can just mock `ConfirmationMessageComponent` with a test component that has the same `@Input` (interface). 

If you want to be super paranoid, you could try to get a reference to the component using `hostCompDe.query(By.directive(TestConfirmationMessageComponent));` to make sure it was used correctly in `ConfirmationDialogComponent`s template (similar to what we do in the tests for `DisplayEditComponent` to make sure the correct value component was selected in the template).

Reply:
I mocked the `ConfirmationMessageComponent` in d9cde94 but I'm having some issues using `hostCompDe.query(By.directive(MockConfirmationMessageComponent));` to check if it exists.

Reply:
I can try to help with that on Monday. We are having a meeting anyway.